arcusApp/app/src/main/java/arcus/app/subsystems/climate/controllers/ThermostatCardController.java

```diff
@@ -227,16 +226,6 @@ public void show(@NonNull ThermostatProxyModel model) {
                 deviceCard.setBottomButtonEnabled(true);
             }
             mMode = model.getMode();
-
-            //need to disabled controls for honeywell device if there are other error conditions.
-            if(model.isHoneywellDevice() && ((deviceCard.isOffline() && deviceCard.isCloudDevice()) ||
```

Comment:
maybe check for cloud without the other checks? /shrug

Reply:
The comment seems to say that this case is specific to the Honeywell device.
Should all cloud thermostats have their controls disabled?

Reply:
ah yeah, let's just kill this for now. Not sure what these left/right/bottom buttons are anyway.